libr/util/sys.c

```diff
@@ -905,6 +935,55 @@ R_API int r_sys_run(const ut8 *buf, int len) {
 	return ret;
 }
 
+R_API int r_sys_run_rop(const ut8 *buf, int len) {
+	const int sz = 4096;
+	int pdelta;
+#if USE_FORK
+	int st, pid;
+#endif
+	// TODO: define R_SYS_ALIGN_FORWARD in r_util.h
+	ut8 *bufptr, *p = malloc ((sz + len) << 1);
```

Comment:
chk for null if allocation fails

Reply:
Use calloc. You are allocating more than what its set via memcoy later which can result on some warnings on static analysis tools or ub if i missed something

Reply:
To be honest this part was left in when I copied the `r_sys_run` function, I'll remove the stack space allocation that I accidentally left in, didn't realize it was there hah